docs/tutorials/podman_tutorial.md

```diff
@@ -140,10 +141,11 @@ podman ps
 Note: If you add *-a* to the *ps* command, Podman will show all containers.
 ### Inspecting a running container
 You can "inspect" a running container for metadata and details about itself.  We can even use
-the inspect subcommand to see what IP address was assigned to the container.
+the inspect subcommand to see what IP address was assigned to the container (none):
```

Comment:
can we say "(none, as rootless containers do not receive an IP address)"?

Reply:
I'd prefer making it a new sentence, but we certainly should add a bit more than '(none)' here.  I'd suggest something like:
"As the container is running in rootless mode, an IP address is not assigned and the value will be listed as "none" in the output from inspect."

Reply:
Yes, when the tutorial was first created, rootless was not an option at that time.